.github/workflows/CICD.yml

```diff
@@ -33,11 +32,26 @@ jobs:
       with:
         command: fmt
         args: -- --check
-    - name: Run clippy (on minimum supported rust version to prevent warnings we can't fix)
+    - name: Ensure MSRV is set in `clippy.toml`
+      run: grep "^msrv = \"${{ env.MIN_SUPPORTED_RUST_VERSION }}\"\$" clippy.toml
+    - name: Run clippy
```

Comment:
Is there a way to pass `msrv` via the command line to clippy? This way, we could get rid of the chek and we wouldn't need to clutter our root directory.

Reply:
I think we can have it as

```rust
#![clippy::msrv = "1.30.0"]
```

Reply:
Good point. Thank you.